Dockerfile.e2e

```diff
@@ -2,7 +2,7 @@
 # with the cluster logging operator source so we can use all the
 # test scripts to deploy the elasticsearch operator and then
 # test the clusterlogging operator
-FROM registry.svc.ci.openshift.org/ocp/4.6:elasticsearch-operator-src
+FROM registry.svc.ci.openshift.org/ocp/4.7:elasticsearch-operator-src
```

Comment:
FYI, this is actually the blocker that requires us to merge this manually.

Reply:
reverted to 4.6 for now... should matter for CLO